TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Persons/Index.cshtml

```diff
@@ -19,7 +20,16 @@
                         <govuk-input-hint class="moj-search__hint">Search by: Name, TRN or date of birth (DD/MM/YYYY)</govuk-input-hint>
                     </govuk-input>
                     <govuk-button class="moj-search__button" type="submit">Search</govuk-button>
-                </div>                
+                </div>
+                <govuk-select asp-for="SortBy">
+                    <govuk-select-label>Sort by</govuk-select-label>
```

Comment:
Could we use `[Display(Name = "Sort by")]` on the model property instead of explicit label tag?

Reply:
Done